setup.cfg

```diff
@@ -0,0 +1,4 @@
+[flake8]
+# 201 and 202 are spaces after ( and before ), 501 is line length. These
+# are exceptions allowed (encouraged?) by Galaxy style guidelines.
```

Comment:
@erasche 

contributing.md explicitly mentions the style guidelines and links to them:

"If your changes modify code - please ensure the resulting files conform to Galaxy [style guidelines](https://wiki.galaxyproject.org/Develop/BestPractices)."